### ISSUE TITLE
Latest available version message isn't reporting anything useful

### ISSUE THREAD

**Issue by tracywalker** [mozilla/iris_firefox#4059]
At test run, just after Iris fetches build information there is a line about latest available version.  I believe this line is just reporting the month. 
ie:
`Latest available version for nightly channel is: 10`

In September the same line was reporting `09`

Should this be reporting a full build id?

**Comment by mwobensmith** [comment]
@tracywalker How important is this debug info? I am looking at this, and it does some weird string parsing to try to account for all of the various formats of the build path. I can write code that deals with the few cases I've been able to surface, but I have a nagging feeling that there will be something unexpected - like other channels, dot release numbers, etc. that will mess up this parsing logic and cause it to blow up.

In other words, it'd be easier to just disable this line rather than risk getting it wrong. It just depends on your needs here.

**Comment by tracywalker** [comment]
Disable it is fine.  whatever it's trying to report isn't really important.  The line just above it for the "Selected build' is probably all we really need to know.

**Comment by mwobensmith** [comment]
Found a different way to approach this, hope it's robust. If it gives us any problems in the future, we'll disable it.
